conf/messages.en

```diff
@@ -239,7 +239,7 @@ cf.authorities.request.received.link = Back to manage import duties and VAT acco
 cf.authorities.notification-panel.p1=The list of authorities you requested are available to download:
 cf.authorities.notification-panel.a.gb-authority=GB authorities (CSV)
 cf.authorities.notification-panel.a.xi-authority=Northern Ireland authorities (CSV)
-cf.authorities.notification-panel.p2=Your list was created on the {0}. It will be available to view for 10 days.
+cf.authorities.notification-panel.p2=Your list was created on {0}. It will be available to view for 10 days.
```

Comment:
For the English message, existing is 'Your list was created on 25 November 2022'

But it should be 'Your list was created on 19th August 2022.' as per ticket. 
Could you please confirm with Sujay?

Reply:
the date is dynamic @tyagikumarvineet

Reply:
@wg-hmrc yes, could saw that earlier  as well that the date is dynamic but my query is whether it is correct cause as per ticket it should be 'Your list was created on 19th August 2022.' And it should be the same for both English and Welsh. In Welsh its static. Have you confirmed with Sujay?

Reply:
@wg-hmrc I have discussed it with Sujay and date in Welsh translation should also be dynamic. He would raise a separate ticket to update the Welsh translation date to be dynamic. Hence I am resolving this